clients/go/test/util/public_key_server.go

```diff
@@ -15,52 +15,27 @@
 package util
 
 import (
-	"crypto/x509"
-	"encoding/json"
-	"encoding/pem"
 	"fmt"
-	"log"
 	"net/http"
 	"net/http/httptest"
-	"strings"
-
-	"github.com/lestrrat-go/jwx/v2/jwk"
+	"os"
 )
 
-// Matching private key here: https://github.com/abcxyz/lumberjack/blob/92782c326681157221df37e0897ba234c5a22240/integration/testrunner/grpcrunner/grpc.go#L60
-const publicKeyString = `
------BEGIN PUBLIC KEY-----
-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEhBWj8vw5LkPRWbCr45k0cOarIcWg
-ApM03mSYF911de5q1wGOL7R9N8pC7jo2xbS+i1wGsMiz+AWnhhZIQcNTKg==
------END PUBLIC KEY-----
-`
+func readBytes() ([]byte, error) {
+	b, err := os.ReadFile("test_public_key.key")
+	if err != nil {
+		return nil, fmt.Errorf("faile to read from file: %w", err)
+	}
+	return b, nil
+}
 
 // StartLocalPublicKeyServer parse pre-made key and set up a server to host it in JWKS format.
-// This is intended to stand in for the JVS in the integration tests.
 func StartLocalPublicKeyServer() (string, func(), error) {
-	block, _ := pem.Decode([]byte(strings.TrimSpace(publicKeyString)))
-	key, err := x509.ParsePKIXPublicKey(block.Bytes)
+	j, err := readBytes()
```

Comment:
Since readBytes is much simpler now, is it still needed, should we just read file here?

Reply:
I think that's a good idea, just updated the code.